airbyte-integrations/connectors/source-sendgrid/integration_tests/invalid_time.json

```diff
@@ -1,4 +1,4 @@
 {
-  "apikey": "apikey",
-  "start_time": "some erroneous input"
+  "api_key": "apikey",
+  "start_date": "some erroneous input"
```

Comment:
Wouldn't it be better for users to update the config field name in the manifest, so it matches the old one? Then the change isn't quite so breaking. The builder hardcodes certain conventional user input names when generating the manifest, but if you manually edit them and paste it back into the the yaml view, it will handle different ones just fine.

Reply:
I'm trying to get on the builder train 🚂 as much as possible. If the builder wants to let me change this, then I'm down with it and that's a good idea. But maintainability means sticking with the tools. If there is a breaking change to make, I want to make it sooner than later.

Reply:
>  But maintainability means sticking with the tools. If there is a breaking change to make, I want to make it sooner than later.

Do we plan to use the builder to maintain connectors in the future?